### PR TITLE
Auto OCP-21814

### PR DESCRIPTION
**Pull request by huiran0826**
Auto SDN case OCP-21814.

Any comments will be welcomed, thanks!
@zhaozhanqi @anuragthehatter @lihongan

logs:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/44884/console

### CODE REVIEW

File: features/networking/service.feature [hunk 1]
```diff
@@ -129,3 +129,27 @@ Feature: Service related networking scenarios
...
+    """
+    When I run the :get client command with:
+      | resource      | pod                     |
+    Then the step should succeed
+    And the output should contain 2 times:
+      | ImagePullBackOff |
+    """
+
+    When I run the :get client command with:
+     | resource      | ep          |
```
Comment (zhaozhanqi) [inline features/networking/service.feature]: seems lack one space for indentation, other LGTM

**Comment by zhaozhanqi** [comment]
LGTM
/assign @akostadinov

**Comment by lihongan** [comment]
LGTM. thanks

**Pull request merged**